civis/io/_tables.py

```diff
@@ -819,24 +858,29 @@ def csv_to_civis(filename, database, table, api_key=None, client=None,
                                   sortkey1=sortkey1, sortkey2=sortkey2,
                                   delimiter=delimiter, headers=headers,
                                   credential_id=credential_id,
+                                  primary_keys=primary_keys,
+                                  last_modified_keys=last_modified_keys,
+                                  escaped=escaped, execution=execution,
                                   polling_interval=polling_interval,
                                   hidden=hidden)
     return fut
 
 
-def civis_file_to_table(file_id, database, table, client=None,
+def civis_file_to_table(file_ids, database, table, client=None,
                         max_errors=None, existing_table_rows="fail",
                         diststyle=None, distkey=None,
                         sortkey1=None, sortkey2=None,
-                        delimiter=",", headers=None,
+                        primary_keys=None, last_modified_keys=None,
+                        escaped=False, execution="immediate",
+                        delimiter=None, headers=None,
                         credential_id=None, polling_interval=None,
                         hidden=True):
```

Comment:
Is there any case when it would be helpful to expose options for `files.post_process_csv` such as `force_character_set_conversion`?

Reply:
I don't think so. These options are set the way they are in order to achieve parity with what `AutoImport`s were/are doing under the hood. Exposing them to users would add to the complexity of an already-massive function signature for little benefit. If there's a specific use case in mind, then we should discuss further, but if it's just for the sake of having another parameter, then in my opinion we don't gain enough to justify the cost.

Reply:
Awesome, thanks!